classes/class-core.php

```diff
@@ -534,6 +534,11 @@ public function enqueue_admin_stylescripts() {
 		
 		if( false !== strpos( $screen->base, 'calderawp_license_manager' ) ){
 
+			//Add CF Core jQuery 1.12 script
+			if( defined( 'CFCORE_URL' ) && file_exists(CFCORE_PATH . "assets/build/js/jquery-12-4.min.js") ) {
+				wp_enqueue_script( "jqueryOneTwelve-license-manager",  CFCORE_URL . "assets/build/js/jquery-12-4.min.js");
```

Comment:
@New0 This file may not exist -- IE if user isn't on latest CF.

Probably the best solution is to not load the plugin if we 5.6 or later and CF 1.9.2

Reply:
Thanks, @Shelob9 I was first tempted to check against version numbers and then thought I would check if file_exists in case we remove it in a later version...
It's now checking against versions and file_exists by https://github.com/CalderaWP/calderawp-license-manager/pull/16/commits/8c99248c3927066249ff1ab8f82ae16006f0b2ae